250215 전가배 9문제 solve complete!

jeongabae opened the pull request:
일단 8문제 올림니다...

jeongabae committed:
first commit

kimregular commented on src/w250210/가배/p16236_아기상어/Main.java:
```diff
@@ -0,0 +1,84 @@
+            int x = cur[0], y = cur[1], dist = cur[2];
+            //물고기 있는 경우, 우선순위에 따라 먹을 물고기 선택(거리/위쪽/왼쪽 우선)
+            if (0 < arr[x][y] && arr[x][y] < curSize) {
+                if (dist < minDist || (dist == minDist && (x < minX || (x == minX && y < minY)))) {
```
물고기 우선순위 구하는 로직을 이렇게 할 수도 있네요! 멋져요

kimregular reviewed (approved):
코드를 잘 짜면 주석이 없어도 이해하기 쉽다,,, 👍

jeongabae committed:
[DFS]준환이의 양팔저울 D4
https://swexpertacademy.com/main/code/problem/problemDetail.do?contestProbId=AWAe7XSKfUUDFAUw

mjbaa commented on src/w250210/가배/p16236_아기상어/Main.java:
물고기 찾는 로직이 엄청 깔끔하네요..보고 배우겠습니다

rocher71 commented on src/w250210/가배/p16236_아기상어/Main.java:
```diff
@@ -0,0 +1,84 @@
+            }
+        }
+
+        System.out.println(time);
```
코드 잘봤습니다 가독성이 좋아서 잘 읽혔어요ㅎㅎ!

혹시 그럴 경우는 없겠지만 만약 fishList가 null 이어서 break 되면 time 은 0이 출력되는게 맞나요?!
